### PR TITLE
modified doc string of memmodel/linear_model.py 

### PR DESCRIPTION
**Pull request by abhsharma2**
Changed docstrings of memmodel/linear_model.py to include code examples

### CODE REVIEW

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (oualib) [inline verticapy/machine_learning/memmodel/linear_model.py:46]: please use multiple block for each operation.

For example:

A linear model is defined by its coefficients and an intercept value. In this example, we will use the following.

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (oualib) [inline verticapy/machine_learning/memmodel/linear_model.py:47]: Let's create a LinearModel.

New block. 

Having many blocks makes the code clear and understandable.

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (oualib) [inline verticapy/machine_learning/memmodel/linear_model.py:49]: Create a dataset.
```
data = [[1.0, 0.3], [2.0, -0.6]]
```
Do some prediction:
```
model_lm.predict(data)
```
Give also example on other methods.

You can separate each method used with a title example:

Making Predictions
^^^^^^^^^^^^^^

Add some notes and hints.

Example:

.. hint:: This object can be pickled and used in any in-memory environment, just like SKLEARN models.

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (oualib) [inline verticapy/machine_learning/memmodel/linear_model.py:202]: Same comment as ABOVE.

**Review by oualib — CHANGES_REQUESTED** [review]
Please:
 - add more example (separated by section)
 - format properly your code
 - use multiple blocks
 - add more notes and hints

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (oualib) [inline verticapy/machine_learning/memmodel/linear_model.py:242]: Same as ABOVE. Create here two subsections:

1. Making in-Memory Prediction -> predict & predict_proba
2. Deploy SQL Code -> predict_sql & predict_proba_sql

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (oualib) [inline verticapy/machine_learning/memmodel/linear_model.py:292]: is it not better each time to execute using ipython cells?

**Review by oualib — CHANGES_REQUESTED** [review]
some corrections to make

**Comment by oualib** [comment]
@mail4umar as all these objects are classes, do you have any best practices to display them nicely?

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (abhsharma2) [reply verticapy/machine_learning/memmodel/linear_model.py:49]: done

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (abhsharma2) [reply verticapy/machine_learning/memmodel/linear_model.py]: done

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (abhsharma2) [reply verticapy/machine_learning/memmodel/linear_model.py:82]: Done

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (abhsharma2) [reply verticapy/machine_learning/memmodel/linear_model.py:98]: Changed each code block to be ipython cell. Output is rendered inside each cell now.
Check the rendered html at http://10.20.71.92:8000/

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (abhsharma2) [reply verticapy/machine_learning/memmodel/linear_model.py:242]: Done

**Review by oualib — APPROVED** [review]
Please wait for Umar's approval before merging.

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (mail4umar) [inline verticapy/machine_learning/memmodel/linear_model.py:34]: Perhaps, we can change the InMemoryModel to ``InMemoryModel`` as this is a specific term for our module.

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (mail4umar) [inline verticapy/machine_learning/memmodel/linear_model.py:39]: same goes for memModel -> ``memModels``

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (mail4umar) [inline verticapy/machine_learning/memmodel/linear_model.py:1]: The main page looks nice. But it is missing ``predict_proba`` and ``predict_proba_sql`` function examples.

And these examples can either be directly under the class as well as the specific function. For example you have an example for predict under the main doc string of the class. You can copy it under Examples heading inside the ``predict`` function as well.

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (mail4umar) [inline verticapy/machine_learning/memmodel/linear_model.py:99]: just like `SKLEARN <https://scikit-learn.org/>`_

**Review by mail4umar — CHANGES_REQUESTED** [review]

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (abhsharma2) [reply verticapy/machine_learning/memmodel/linear_model.py:34]: done

File: verticapy/machine_learning/memmodel/linear_model.py
Comment (abhsharma2) [reply verticapy/machine_learning/memmodel/linear_model.py:1]: added both predict_proba and predict_proba_sql examples under `LinearModelClassifier` as they are not applicable to linear regression model

**Comment by mail4umar** [comment]
@abhsharma2 you can update the changes and then run the checks.

**Pull request merged**
